### PR TITLE
Fix bad translation string

### LINKED ISSUE #37730: Unable to translate string "{0} database {1} default timezone is {2}" to fr
### Describe the bug

With version 0.48.3, I have this error message in the logs

### To Reproduce

See log

### Expected behavior

_No response_

### Logs

```
janv. 16 09:15:12 xxxxxx metabase[101054]: 2024-01-16 09:15:12,175 ERROR i18n.impl :: Unable to translate string "{0} database {1} default timezone is {2}" to fr
janv. 16 09:15:12 xxxxxx metabase[101054]: java.lang.IllegalArgumentException: Unmatched braces in the pattern.
janv. 16 09:15:12 xxxxxx metabase[101054]:         at java.base/java.text.MessageFormat.applyPattern(MessageFormat.java:520)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at java.base/java.text.MessageFormat.<init>(MessageFormat.java:390)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n.impl$message_format.invokeStatic(impl.clj:166)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n.impl$message_format.invoke(impl.clj:164)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n.impl$translate.invokeStatic(impl.clj:195)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n.impl$translate.invoke(impl.clj:170)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n$translate_site_locale.invokeStatic(i18n.clj:73)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n$translate_site_locale.invoke(i18n.clj:70)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.util.i18n.SiteLocalizedString.toString(i18n.clj:98)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.core$str.invokeStatic(core.clj:555)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.core$str.invoke(core.clj:546)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.sync_metadata.sync_timezone$fn__80297$_AMPERSAND_f__80298.invoke(sync_timezone.clj:42)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.sync_metadata.sync_timezone$fn__80297$fn__80303.invoke(sync_timezone.clj:36)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.lang.AFn.applyToHelper(AFn.java:154)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.lang.AFn.applyTo(AFn.java:144)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.core$apply.invokeStatic(core.clj:669)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.core$apply.invoke(core.clj:662)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57415$_AMPERSAND_f__57417$fn__57419.doInvoke(util.clj:425)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.lang.RestFn.invoke(RestFn.java:397)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:132)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:126)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_debug_logging.invokeStatic(util.clj:150)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_debug_logging.invoke(util.clj:146)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57415$_AMPERSAND_f__57417.invoke(util.clj:420)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57415$fn__57424.invoke(util.clj:415)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57514$_AMPERSAND_f__57515$fn__57523.invoke(util.clj:531)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57514$_AMPERSAND_f__57515.invoke(util.clj:529)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57514$fn__57532.invoke(util.clj:523)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.sync_metadata$fn__82263$_AMPERSAND_f__82264$fn__82265.invoke(sync_metadata.clj:63)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:191)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$do_with_error_handling.invoke(util.clj:184)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.core$partial$fn__5910.invoke(core.clj:2647)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.driver$fn__48862.invokeStatic(driver.clj:725)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.driver$fn__48862.invoke(driver.clj:725)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at clojure.lang.MultiFn.invoke(MultiFn.java:239)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$sync_in_context$fn__57348.invoke(util.clj:167)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_db_logging_disabled$fn__57345.invoke(util.clj:159)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:132)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:126)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_start_and_finish_logging$fn__57332.invoke(util.clj:144)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57319$_AMPERSAND_f__57320$fn__57324.invoke(util.clj:118)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$with_duplicate_ops_prevented$fn__57309.invoke(util.clj:90)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57363$_AMPERSAND_f__57364.invoke(util.clj:216)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.util$fn__57363$fn__57366.invoke(util.clj:210)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.sync_metadata$fn__82263$_AMPERSAND_f__82264.invoke(sync_metadata.clj:62)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.sync.sync_metadata$fn__82263$fn__82268.invoke(sync_metadata.clj:58)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.task.sync_databases$sync_and_analyze_database_STAR__BANG_.invokeStatic(sync_databases.clj:94)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.task.sync_databases$sync_and_analyze_database_STAR__BANG_.invoke(sync_databases.clj:77)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.task.sync_databases$sync_and_analyze_database_BANG_.invokeStatic(sync_databases.clj:113)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.task.sync_databases$sync_and_analyze_database_BANG_.invoke(sync_databases.clj:101)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at metabase.task.sync_databases.SyncAndAnalyzeDatabase.execute(sync_databases.clj:118)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
janv. 16 09:15:12 xxxxxx metabase[101054]:         at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
janv. 16 09:15:12 xxxxxx metabase[101054]: 2024-01-16 09:15:12,201 INFO sync-metadata.sync-timezone :: :mysql database 6 default timezone is "CET"

```

### Information about your Metabase installation

```JSON
{
  "browser-info": {
    "language": "fr",
    "platform": "Linux x86_64",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.15+10-post-Debian-1deb11u1",
    "java.vendor": "Debian",
    "java.vendor.url": "https://tracker.debian.org/openjdk-11",
    "java.version": "11.0.15",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.15+10-post-Debian-1deb11u1",
    "os.name": "Linux",
    "os.version": "5.10.0-14-amd64",
    "user.language": "fr",
    "user.timezone": "Europe/Paris"
  },
  "metabase-info": {
    "databases": [
      "mysql"
    ],
    "hosting-env": "unknown",
    "application-database": "mysql",
    "application-database-details": {
      "database": {
        "name": "MariaDB",
        "version": "10.5.15-MariaDB-0+deb11u1"
      },
      "jdbc-driver": {
        "name": "MariaDB Connector/J",
        "version": "2.7.10"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2024-01-11",
      "tag": "v0.48.3",
      "hash": "80d8323"
    },
    "settings": {
      "report-timezone": "Europe/Paris"
    }
  }
}
```

### Severity

little

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by calherries**
Fixes https://github.com/metabase/metabase/issues/37730

Removes the translation of the problematic log message.

The translated french string is wrong anyway, since the format string args don’t line up. The french string has 2 args and the english has 3:
```
en: {0} database {1} default timezone is {2}
fr: Le fuseau horaire par défaut de la base de données {0} est {1}
```
Since we’re okay removing the translation of log messages in 49, I assume it’s also okay in 48 to remove translating this log message.

### CODE REVIEW

**Review by sansolovyov — APPROVED** [review]

File: locales/fr.po [hunk 1]
```diff
@@ -1764,7 +1764,7 @@ msgstr "Précisément, quand vous intégrez des questions ou des tableaux de bor
 msgid "Enable"
 msgstr "Activer"
 
-#. Que pensez-vous de "intégrer" / "intégration" / etc ?
+#. Que pensez-vous de "intégrer" / "intégration" / etc ?
```
Comment (calherries) [inline locales/fr.po:1767]: This was automatically done by my editor trimming whitespace from the end of the file. It's just a comment, so it shouldn't matter, but to be sure...

With whitespace highlighting on:
```
git config diff.wsErrorHighlight all
```
the git diff is:
```diff
diff --git a/locales/fr.po b/locales/fr.po
index 9142c585f8..fc28db5d69 100644
--- a/locales/fr.po
+++ b/locales/fr.po
@@ -1764,7 +1764,7 @@ msgstr "Précisément, quand vous intégrez des questions ou des tableaux de bor
 msgid "Enable"
 msgstr "Activer"
 
-#. Que pensez-vous de "intégrer" / "intégration" / etc ?^M
+#. Que pensez-vous de "intégrer" / "intégration" / etc ?
 #. (pour "embed", "embedding", etc)
 #: frontend/src/metabase/admin/settings/components/widgets/EmbeddingLevel.jsx:28
 msgid "Premium embedding enabled"
@@ -30398,7 +30398,7 @@ msgstr "Fuseau horaire incorrect {0} : {1}"
 
...
```

`^M` indicates the carriage return character. It was written with a windows computer.

Comment (piranha) [reply locales/fr.po:1767]: Interesting that there is only one such line in the whole file.

**Comment by calherries** [comment]
I realised the translated french string is wrong anyway, since the format string args don’t line up. The french string has 2 args and the english has 3:
```
en: {0} database {1} default timezone is {2}
fr: Le fuseau horaire par défaut de la base de données {0} est {1}
```

Since we’re okay removing the translation of log messages in 49, I assume it’s also okay in 48 to remove translating this log message.

**Review by sansolovyov — APPROVED** [review]
That's even better!

**Pull request merged**